examples/hotel-bookings/src/index.md

```diff
@@ -0,0 +1,264 @@
+  </div>
+  <div class="card grid-colspan-2 grid-rowspan-2">
+  <h2>Rooms reserved by season</h2>
+    ${resize((width, height) => typeSeasonBubble(width, height))}
+  </div>
+  <div class="card grid-colspan-2 grid-rowspan-2">
+    ${resize((width, height) => dailyRateChart(width, height))}
+  </div>
+</div>
+
+<div class="card" style="padding: 0">
+${display(Inputs.table(cleanTable))}
+</div>
+
+```js
+// Line chart (arrival dates)
+function arrivalLineChart(width, height) {
+  return Plot.plot({
+    height: height - 50,
+    marginBottom: 35,
+    width,
+    x: { label: "Arrival date" },
+    y: { label: "Bookings", grid: true },
+    title: `${pickMarketSegment} bookings by arrival date`,
+    caption: "Daily counts and 28-day moving average.",
+    marks: [
+      Plot.lineY(
+        bookingsByMarketSegment,
+        Plot.binX(
+          { y: "count" },
+          {
+            x: "arrivalDate",
+            interval: d3.utcDay,
+            opacity: 0.5,
+            strokeWidth: 1
+          }
+        )
+      ),
+      Plot.lineY(
+        bookingsByMarketSegment,
+        Plot.windowY(
+          { k: 28 },
+          Plot.binX(
+            { y: "count" },
+            {
+              x: "arrivalDate",
+              interval: d3.utcDay,
+              strokeWidth: 2,
+              tip: true
+            }
+          )
+        )
+      ),
+      Plot.ruleY([0]),
+      Plot.axisX({ ticks: 5 }),
+      Plot.axisY({ ticks: 5 })
+    ],
+  });
+}
+```
+
+```js
+// Booking lead time bar chart
+function leadTimeChart(width, height) {
+  return Plot.plot({
+    width,
+    height: height - 30,
+    x: { label: "Booking lead time (days)" },
+    y: { grid: true },
+    color: { legend: true },
+    marks: [
+      Plot.ruleY([0]),
+      Plot.axisX({ ticks: 5 }),
+      Plot.axisY({ ticks: 5 }),
+      Plot.rectY(
+        bookingsByMarketSegment,
+        Plot.binX(
+          { y: "count" },
+          { x: "LeadTime", interval: 20, fill: "season" }
+        )
+      ),
+    ],
+  });
+}
+```
+
+```js
+// Daily rate stacked histogram
+function dailyRateChart(width, height) {
+  return Plot.plot({
+    width,
+    height: height - 30,
+    x: { label: "Daily rate" },
+    y: { grid: true },
+    color: { legend: true },
+    marks: [
+      Plot.ruleY([0]),
+      Plot.axisX({ ticks: 5 }),
+      Plot.axisY({ ticks: 5 }),
+      Plot.rectY(
+        bookingsByMarketSegment,
+        Plot.binX(
+          { y: "count" },
+          { x: "ADR", fill: "#21C6A8", interval: 20, fill: "season", tip: true}
+        )
+      ),
+    ],
+  });
+}
+```
+
+**Data source:** Antonio et al (2021). Hotel booking demand datasets. Data in Brief (22): 41-49. https://doi.org/10.1016/j.dib.2018.11.126
+
+**Credit:** Donut charts reuse code from [Mike Bostock's _Donut chart_ notebook](https://observablehq.com/@d3/donut-chart/2). Big number boxes are adapted from [Paul Buffa’s _Walmart growth dashboard_](https://observablehq.com/d/a156c1245672f9bd).
```

Comment:
I don't think we should credit our own notebooks
```suggestion
```

Reply:
Removed